modules/assim.batch/R/pda.load.data.R

```diff
@@ -38,13 +40,19 @@ load.pda.data <- function(settings, con) {
     
     vars.used.index <- which(format$vars$bety_name %in% c(inputs[[i]]$variable.name))
     
+    if(format$file_name == "AmeriFlux.level2.h.nc"){
+      time.row <- NULL
+    }else{
+      time.row <- which(format$vars$bety_name %in% c("year", "month", "week", "day", "hour", "minute", "second"))
+    }
```

Comment:
This shouldn't be necessary. First, the query to format should give you the time row automatically. Second, we definitely want to avoid format-specific hacks. Third, the else case will probably fail a lot because it will miss other possible time columns -- it doesn't make sense to rely on catching any possible column name when @bcow already has a way to catch time columns based on time formatting.